native/packages/react-native-bpk-component-button/src/BpkButton-test.common.js

```diff
@@ -26,8 +26,7 @@ import BpkButton, { propTypes, BUTTON_TYPES } from './BpkButton';
 
 const onPressFn = jest.fn();
 const commonTests = () => {
-  jest.mock('Image', () => 'Image');
-
+  jest.mock('Text', () => 'Text');
```

Comment:
Why is this line indented like this?

Reply:
2 spaces indent because it's in a function - I don't see the issue?

![screen shot 2017-10-19 at 11 34 47](https://user-images.githubusercontent.com/73652/31767032-947c8e06-b4c1-11e7-8e75-39ba932d7dc8.png)

Reply:
Hmm sorry, I'm blind it seems

Reply:
It's ok, the glasses you're wearing in your photo do look quite dark 😆

Reply:
![image](https://user-images.githubusercontent.com/1333960/31779367-83fadd64-b4e2-11e7-9aec-d3fa9f1a4a51.png)